ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/Offset.scala

```diff
@@ -21,22 +21,24 @@ import com.google.protobuf.ByteString
   * less than newer offsets.
   *
   */
-final class Offset(private val value: ByteString) extends AnyVal with Ordered[Offset] {
-
+case class Offset(bytes: Bytes) extends Ordered[Offset] {
```

Comment:
Hmmm… can this be `AnyVal`?

Reply:
It cannot because Bytes is already a user-defined `AnyVal`

Reply:
It cannot, because `Bytes` is already `AnyVal`.

Reply:
I did not know that was a thing.

_Samir has learnt something today._